scripts/start-local.sh

```diff
@@ -11,12 +11,16 @@ docker compose -f "${DOCKER_COMPOSE_FILE}" stop
 docker rm fake-manage-recalls-api || true
 docker build fake-manage-recalls-api
 
+docker rm fake-prison-register-api || true
+docker build fake-prison-register-api
```

Comment:
You don't need to do a `rm`/`build` here if you're pulling from a pre-built image.

Reply:
Got it, thanks! Will remove

Reply:
Remove the `build` step too 😉